downstream/modules/eda/proc-eda-activate-webhook.adoc

```diff
@@ -0,0 +1,73 @@
+[id="proc-eda-activate-webhook"]
+
+= Activating webhook rulebooks
+
+Rulebooks that include a `source` section require extra configuration to accept external messages as triggers, usually in the form of a webhook. 
```

Comment:
This could not be exact, the sources doesn't require necessarily to accept external messages, only for the webhook plugin or any other that includes a "port" param.  In fact, according to the best practices promoted by @matburt the webhook plugin would be the less recommended pattern. Ref: https://ansible.readthedocs.io/projects/rulebook/en/latest/sources.html#best-practices-and-patterns